### PR TITLE
SMS > Receive an SMS (PHP)

### LINKED ISSUE #146: Developer Beta for receiving sms there is no php. Just Ruby and json. Please include php. Everyone understand php better

### PR DESCRIPTION
**Pull request by adambutler**
## Description

Adds PHP example for Receive an SMS. Fixes #146 

## Deploy Notes

None

### CODE REVIEW

**Comment by adambutler** [comment]
@leggetter This may need some more surrounding content from a PHP dev as per the Node.js / Ruby examples.

File: .repos/nexmo-community/nexmo-php-quickstart/sms/receive-sms.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php 
+require_once __DIR__ . '/../config.php';
+
+require_once __DIR__ . '/../vendor/autoload.php';
```
Comment (leggetter) [inline .repos/nexmo-community/nexmo-php-quickstart/sms/receive-sms.php]: Is there a way to include this, but not show rows 1 to 5?

Comment (adambutler) [inline .repos/nexmo-community/nexmo-php-quickstart/sms/receive-sms.php]: Yep 👍  On it.

**Comment by adambutler** [comment]
@mheap Mind taking a look at adding the surrounding copy here please.

**Comment by adambutler** [comment]
@leggetter Looks like @mheap is away this week... mind taking a look at the copy here?

**Comment by adambutler** [comment]
@mheap Is this ready for review? If so can you bring up-to-date with master and let me know.

**Pull request closed without merging**